addons/html_builder/static/src/plugins/utils.js

```diff
@@ -1,10 +1,13 @@
-export function applyFunDependOnSelectorAndExclude(fn, rootEl, selector, exclude) {
+export function applyFunDependOnSelectorAndExclude(fn, rootEl, selector, exclude, applyTo) {
```

Comment:
Here we should probably use an object for the parameters as `exclude` and `applyTo` can be not set and we want to differentiate those two :+1: